analysis/network_models.py

```diff
@@ -19,15 +19,137 @@
 # Estimated size of a http response header in bytes for a HTTP2 response using HPACK.
 ESTIMATED_HTTP_RESPONSE_HEADER_SIZE = 35
 
-# TODO(garretrieger): populate with some real network models.
-BROADBAND = simulation.NetworkModel(
-    "broadband",
-    rtt=20,  # 40 ms
-    bandwidth_up=1250,  # 10 mbps
-    bandwidth_down=6250)  # 50 mbps
-
-DIALUP = simulation.NetworkModel(
-    "dialup",
-    rtt=200,  # 200 ms
-    bandwidth_up=7,  # 56 kbps
-    bandwidth_down=7)  # 56 kbps
+# Mobile 2G
```

Comment:
Would be nice to comment the file with a note about the data source / why we believe this correct